RIAssigner/compute/Kovats.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Iterable
+from RIAssigner.data.Data import Data
+from .ComputationMethod import ComputationMethod
+
+
+class Kovats(ComputationMethod):
+    """ Class to compute the Kovats retention index. """
+
+    def compute(self, query: Data, reference: Data) -> List[float]:
+        """ Compute non-isothermal Kovats retention index.
+        For details see https://webbook.nist.gov/chemistry/gc-ri/.
+
+        Parameters
+        ----------
+        query:
+            Dataset for which to compute retention indices.
+        """
+
+        self._check_data_args(query, reference)
+
+        lower_index = 0
+        higher_index = 0
+        retention_indices = []
+
+        # Copy rts and ris and insert 0 in the beginning, so that interpolation always starts at 0,0 to the first reference compound.
+        reference_rts = list(reference.retention_times)
+        reference_ris = list(reference.retention_indices)
+
+        reference_rts.insert(0, 0.0)

```

Comment:
Can be potentially simplified to 

```python
reference_rts = [0.0] + list(reference.retention_times)
```

Reply:
Doesn't this cause 0 to be added to every element? I'll try and see if it works. Thanks for the hint, didn't know that.

Reply:
```
>>> ['a','b']+[['c']]+[0.0]
['a', 'b', ['c'], 0.0]
```

Reply:
I included the proposed changes and tests are still all green, so seems good.